database/src/postgres/rpc_server.rs

```diff
@@ -2,6 +2,7 @@ use crate::postgres::PostgresStorageManager;
 use crate::AdditionalDatabaseOptions;
 use bigdecimal::ToPrimitive;
 use borsh::{BorshDeserialize, BorshSerialize};
+use std::str::FromStr;
```

Comment:
Please improve the import piece according to the style guides:

1. std goes first
2. External goes next
3. Current crates 
4. Mods

Reply:
Unfortunately I can't change the order of the imports. This is handled by `cargo fmt`.

Reply:
You need to leave an empty line between the groups in that case `fmt` won't reorder it

Reply:
Thank you!